### ISSUE TITLE
Reproducing results

### ISSUE THREAD

**Issue by zskuang58** [jkpark0825/EANet#7]
Firstly, thank you to the authors for open-sourcing the code; it's truly an exciting piece of work. I've encountered a slight issue while reproducing the results of the paper and was hoping the authors could help me clarify this confusion. 

The following presents the results of my experiment reproduction using the author's code without any modifications. These results were obtained during testing at the 21th and 27th epochs, respectively.
![image](https://github.com/jkpark0825/EANet/assets/53181196/4dc3af21-3540-44fc-811b-1959dc983aaa)

It seems to deviate a bit from the results reported in the paper. I would be grateful if the author could help me resolve this contradiction. Here are the results from the paper.
![image](https://github.com/jkpark0825/EANet/assets/53181196/b055b445-5f91-48e6-a907-42e6e7a6c7b9)

**Comment by airparkchen** [comment]
> Firstly, thank you to the authors for open-sourcing the code; it's truly an exciting piece of work. I've encountered a slight issue while reproducing the results of the paper and was hoping the authors could help me clarify this confusion.
> 
> The following presents the results of my experiment reproduction using the author's code without any modifications. These results were obtained during testing at the 21th and 27th epochs, respectively. ![image](https://private-user-images.githubusercontent.com/53181196/296678398-4dc3af21-3540-44fc-811b-1959dc983aaa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5MzUxNTAsIm5iZiI6MTcwODkzNDg1MCwicGF0aCI6Ii81MzE4MTE5Ni8yOTY2NzgzOTgtNGRjM2FmMjEtMzU0MC00NGZjLTgxMWItMTk1OWRjOTgzYWFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI2VDA4MDczMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjYjMyZWFmMWIzODJhMWYwMDk1NDdmNWUzNDA1OGVkNjk3YjJjZTNkZjFlNDk1ZjdjMGJiZjdlNTcwNDI2ZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.F8QGjzdZaHno5Zf4ratr4WrAF-2XyEK6QDlS25uNMLU)
> 
> It seems to deviate a bit from the results reported in the paper. I would be grateful if the author could help me resolve this contradiction. Here are the results from the paper. ![image](https://private-user-images.githubusercontent.com/53181196/296679407-b055b445-5f91-48e6-a907-42e6e7a6c7b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5MzUxNTAsIm5iZiI6MTcwODkzNDg1MCwicGF0aCI6Ii81MzE4MTE5Ni8yOTY2Nzk0MDctYjA1NWI0NDUtNWY5MS00OGU2LWE5MDctNDJlNmU3YTZjN2I5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI2VDA4MDczMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjN2ZlMzM5MWU1OGZhNzdhNGZhYTJlNjY3ZTVhYTRhNjMzY2JiM2U4Nzk5ZDhkMmFjMWZiYjczNTUwNmMxMjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.p_SB34q4IEjcX7fyGEDSp_fz7NFw3FZOzaFQL7nO4Uk)

Hello, I am also trying to retrain this model in hopes of replicating the effects mentioned in the paper. 
I'm using the InterHand2.6M dataset and the code from this repository. However, I only achieved approximately 18mm as a result for the "MPVPE for all Hands" after 21 epochs, which significantly differs from your training results .
May I ask if you made any special modifications to the learning rate, code, or dataset?

**Comment by dqj5182** [comment]
May I ask if the problem still persists?

**Comment by zskuang58** [comment]
Thanks for your reply. It hasn't been solved yet. Can you give me some suggestions
